helm_deploy/values-dev.yaml

```diff
@@ -9,6 +9,7 @@ generic-service:
     API_BASE_URL_OAUTH: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
     API_BASE_URL_NOMIS: "https://api-dev.prison.service.justice.gov.uk"
     API_BASE_URL_RESTRICTED_PATIENTS: "https://restricted-patients-api-dev.hmpps.service.justice.gov.uk"
+    DIFF_HOST: "https://prisoner-offender-search-dev.prison.service.justice.gov.uk"
```

Comment:
As there is no Request (only message in -> message out) we cannot derive the host name.